src/uwtools/tests/test_cli.py

```diff
@@ -588,19 +652,3 @@ def test__parse_args():
         Parser.assert_called_once()
         parser = Parser()
         parser.parse_args.assert_called_with(raw_args)
-
-
-# Helper functions
-
-
-def actions(parser: Parser) -> List[str]:
-    # Return actions (named subparsers) belonging to the given parser.
-    if actions := [x for x in parser._actions if isinstance(x, _SubParsersAction)]:
-        return list(actions[0].choices.keys())
-    return []
-
-
-@fixture
-def subparsers():
-    # Create and return a subparsers test object.
-    return Parser().add_subparsers()
```

Comment:
Moved to top of module and split into `Helpers` and `Fixtures` sections, for consistency with other test modules.